architectures/x86_64/kernel/GDT.h

```diff
@@ -89,6 +89,12 @@ struct __packed GDTEntry64
                      flags,
                      base24_31);
     }
+    GDTEntry64( uint8_t flags, uint8_t gran)
+    {
```

Comment:
this can juste be: `GDTEntry64(uint8_t flags, uint8_t granularity) : GDTEntry64(0, 0xffff, granularit, flags){}`